plugin-server/src/main/pluginsServer.ts

```diff
@@ -198,10 +201,17 @@ export async function startPluginsServer(
             })
         }
 
-        // every minute flush lastSeenAt cache
+        // every 10 seconds past the minute flush lastSeenAt cache
         if (serverConfig.EXPERIMENTAL_EVENTS_LAST_SEEN_ENABLED) {
-            flushLastSeenAtCacheJob = schedule.scheduleJob('0 * * * * *', async () => {
-                await hub!.teamManager.flushLastSeenAtCache()
+            flushLastSeenAtCacheJob = schedule.scheduleJob('10 * * * * *', async () => {
+                await piscina!.broadcastTask({ task: 'flushLastSeenAtCache' })
+            })
```

Comment:
This should be performed on the worker threads. The local teamManager doesn't get any events passing through it. At least it shouldn't.

Reply:
You know better here, don't have all the context to make the right call